data/faq.md

```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
 [EP2021 programme](https://ep2021.europython.eu/events/sessions/) and the
 [EP2019 programme](https://ep2019.europython.eu/events/sessions/).
 
-To select talks, we run a Call for Proposals (CFP). Talks are then curated based
-on the input from the community voting and expert panel reviews. Make sure to subscribe to our
-[EuroPython Blog](https://blog.europython.eu/#/portal/signup) for updates.
+The conference will feature around 20 tutorials/workshops, 120 talk sessions and other interactive sessions such as Posters, Helpdesks, Panel Discussions and Maker Space.
 
-We aim to have a very diverse set of talks, tutorials, workshops, posters and
-keynotes, covering many different angles of where and how Python can be used to
-solve problems or do interesting things.
+We aim to have a very diverse programme, covering many different angles of where and how Python can be used to solve problems or do interesting things.
 
 ## **Q. When do you accept talk proposals?**
 
-A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) opened on 8 March.
+A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) ran between 8 March and 3 April.
+
+We are thrilled to have received a record high of 429 proposals on a diverse range of topics.
+
+## **Q. How do you review and select talks?**
+
+A. We believe that a good conference consists of popular talks as well as new or niche topics - all by and for our diverse community. Thus after Call for Proposals (CFP) finishes, submissions go through two round of unbiased reviews: speakers' names will not be displayed.
```

Comment:
"Thus, "

Hmm... reviews are always biased towards the reviewers point of view. ;-) How about "go through two rounds of review and refinement as we put together our programme. This is a "blind" process, so speakers' names will not be displayed to the reviewers." ..? I use "refinement" in the sense that the programme AND the talks are being refined as stuff gets cut, included or changed. Does that make sense..?

Reply:
Thank you thank you! Makes total sense! I was thinking "unbiased" is a big word! :P